.github/workflows/pr_verification.yml

```diff
@@ -370,6 +370,13 @@ jobs:
           echo "MODULE_CONFIG_HOSTED_CONFIGURATION_MODULE=org.candlepin.hostedtest.AdapterOverrideModule" >> .env
           echo "CANDLEPIN_AUTH_CLOUD_ENABLE=true" >> .env
 
+      - if: matrix.database == 'mariadb'
+        name: Add mysql in use env variable
+        working-directory: ./.github/containers
+        shell: bash
+        run: |
+          echo "MYSQL_IN_USE=true" >> .env
```

Comment:
We don't need this as we are always setting "MYSQL_IN_USE" in mariadb.docker-compose.yaml